cmake/find/blob_storage.cmake

```diff
@@ -1,8 +1,10 @@
+option (ENABLE_AZURE_BLOB_STORAGE "Enable Azure blob storage" ${ENABLE_LIBRARIES})
+
 option(USE_INTERNAL_AZURE_BLOB_STORAGE_LIBRARY
     "Set to FALSE to use system Azure SDK instead of bundled (OFF currently not implemented)"
-    ${ENABLE_LIBRARIES})
+    ON)
```

Comment:
@amosbird Why did you enable blob storage building by default?

Reply:
We don't. Another option is introduced: `ENABLE_AZURE_BLOB_STORAGE`